R/CohortConstruction.R

```diff
@@ -30,8 +30,10 @@ getCohortsJsonAndSqlFromPackage <- function(packageName = packageName,
                               extension = "csv", 
                               add = errorMessage)
   
-  cohorts <- readr::read_csv(pathToCsv, col_types = readr::cols()) %>%
-    dplyr::filter(!.data$cohortId %in% cohortIds)
+  cohorts <- readr::read_csv(pathToCsv, col_types = readr::cols())
+  if (!is.null(cohortIds)) {
+    cohorts <- cohorts %>% dplyr::filter(.data$cohortId %in% cohortIds)
```

Comment:
Note that the `filter` call is using a different logic:

Original: dplyr::filter(**!**.data$cohortId %in% cohortIds)
Revised: dplyr::filter(.data$cohortId %in% cohortIds)

Reply:
Thanks for the fix! @anthonysena